lib/greet.js

```diff
@@ -0,0 +1,17 @@
+'use strict';
+
+module.exports = exports = {};
+var name = process.argv[2];
+
+exports.greet = function(name) {
+  if (arguments.length === 0) throw new Error ('name not included');
+  console.log(`hello ${name}`);
+  return ('hello ' + name);
+
+};
+exports.favPlace = function(place) {
+  if (arguments.length === 0) throw new Error ('place not included');
+  return ('your favorite place is '+ place);
+};
+
+exports.greet(name);
```

Comment:
Since you are already using module.exports at the top of the file, you don't need to explicitly attach the method again to the exports object on line 17. You could just omit line 17 altogether :) I also noticed you are invoking favPlace in index.js but it is expecting an argument since you have passed place as a parameter on line 12. You should pass an argument to the function call in index.js. Something like greet.favPlace('Washington')